sn_node/src/node.rs

```diff
@@ -251,7 +251,7 @@ impl Node {
                             trace!("Periodic bad_nodes check took {:?}", start.elapsed());
                         });
 
-                        if rolling_index == 255 {
+                        if rolling_index == 511 {
```

Comment:
511?

Reply:
because using `% 2` for the `upper / lower part` indexing to half the number of concurrent queries each time.